src/include/cart/api.h

```diff
@@ -407,6 +407,34 @@ crt_reply_get(crt_rpc_t *rpc)
 	return rpc->cr_output;
 }
 
+/**
+ * Return HLC timestamp of RPC was sent
```

Comment:
This comment needs extending, the timestamp of the req_send() function?  What does this mean for RPCs which are being sent, vs RPCs which are being received?

Reply:
done.